dd-sdk-android/src/main/kotlin/com/datadog/android/rum/internal/domain/scope/RumActionScope.kt

```diff
@@ -188,11 +189,14 @@ internal class RumActionScope(
 
         val context = getRumContext()
         val user = CoreFeature.userInfoProvider.getUserInfo()
+        val hasUserInfo = user.id != null || user.name != null ||
+            user.email != null || user.additionalProperties.isNotEmpty()
 
         val frustrations = mutableListOf<ActionEvent.Type>()
         if (trackFrustrations && errorCount > 0 && actualType == RumActionType.TAP) {
             frustrations.add(ActionEvent.Type.ERROR_TAP)
         }
+        val networkInfo = CoreFeature.networkInfoProvider.getLatestNetworkInfo()
```

Comment:
for continuous action network info at the time of sending the action may be not relevant or not the same as it was at the beginning of the action. Like for the `RumResourceScope` we fix the network info when we create resource scope, because this is the state when that was when resource call started.

IMO we should either do the same here, or not add it.